lib/fastlane/setup/setup_ios.rb

```diff
@@ -72,6 +88,27 @@ def generate_appfile
       Helper.log.info "Created new file '#{path}'. Edit it to manage your preferred app metadata information.".green
     end
 
+    def default_run_produce
+      Helper.log.info "Running produce..."
+      require 'produce'
+      config = {}
+      FastlaneCore::Project.detect_projects(config)
+      project = FastlaneCore::Project.new(config)
+
+      produce_options_hash = {
+          app_name: project.default_app_name
+      }
+      Produce.config = FastlaneCore::Configuration.create(Produce::Options.available_options, produce_options_hash)
+      begin
+        ENV['PRODUCE_APPLE_ID'] = Produce::Manager.start_producing
+      rescue Exception => exception
+        Helper.log.info 'It looks like that App Name has already been taken, please enter an alternative.'.yellow
```

Comment:
There can all kinds of exception, the name is just one of them. Easiest way is to add
```ruby
if ex.to_s.include?("bla bla")
```